src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -66,32 +66,35 @@ class CollapsibleCourseRunFields extends React.Component {
         {fields.map((courseRun, index) => (
           <Collapsible
             title={formatCourseRunTitle(courseRuns[index])}
-            iconId={`collapsible-icon-${index}`}
+            key={`collapsible-run-${courseRun}`}
+            iconId={`collapsible-icon-${courseRun}`}
           >
             <Field
               name={`${courseRun}.start`}
               type="date"
               component={RenderInputTextField}
-              format={value => moment(value).format('YYYY-MM-DD')}
+              format={value => (value ? moment(value).format('YYYY-MM-DD') : '')}
               label={
                 <React.Fragment>
                   Start date
                   <span className="required" aria-hidden>*</span>
                 </React.Fragment>
               }
+              placeholder="dd/mm/yyyy"
```

Comment:
The placeholder is `dd/mm/yyyy`, but we format to `YYYY-MM-DD`. Is that just the way it should be viewed vs the way we need to send it to the API?

Reply:
<img width="124" alt="Screen Shot 2019-04-01 at 3 45 50 PM" src="https://user-images.githubusercontent.com/1559107/55355045-3fdfc280-5495-11e9-93fd-d3cfc71dfcfd.png">
Internally it needs that date format, but the display is this

Reply:
I do still need to fix it though :D